src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -87,6 +89,15 @@ const BelowTheFoldArtworkDetails: React.FC<BelowTheFoldArtworkDetailsProps> = ({
 export const ArtworkApp: React.FC<Props> = props => {
   const { artwork, me, referrer, tracking, shouldTrackPageView } = props
   const { match } = useRouter()
+  const hasArtists = (artwork.artists?.length ?? 0) > 0
+  const biddingEndAt =
+    artwork?.saleArtwork?.extendedBiddingEndAt ?? artwork?.saleArtwork?.endAt
+  const { hasEnded } = useTimer(biddingEndAt!, artwork?.sale?.startAt!)
+
+  // TODO: a mix of logic from displaying a "Bidding closed" message from
+  // ArtworkSidebar/ArtworkSidebar and our assumptions
+  const displayAuctionCreateAlertHeader =
+    hasArtists && (hasEnded || lotIsClosed(artwork.sale, artwork.saleArtwork))
```

Comment:
We could check if the FF is enabled here instead of doing it in the ArtworkAuctionCreateAlertHeader.tsx file

Reply:
This was the initial solution but unfortunately the FF was unreachable for testing. We had to add it to the component in order to check if it's enabled or not.

Reply:
In testing, can `jest.mock `useFeatureFlag` to return what you need for test. Did y'all try that? (Also forgive if that's not the actual issue)

Reply:
IMO I think it's better to have the featureflag in one place as @tam-kis did it here, since we will be able to clean up the component in **one place** and we won't have to edit the extra conditional logic here since it is gonna stay after we deploy the change 🤔 

Another reason IMO would be that we write a test for the actual [banner component](https://github.com/artsy/force/pull/12678/files#diff-b4b170cda0cc85103778f704c3ed2baff7037f2310c78b3632a7a886cfc24c24R14-R29)and not have to add another one for the artworkApp

Reply:
Oki, that makes sense @gkartalis
I normally prefer to have the ff on the level where we display the component, not inside as it is here. The reason for it is in my case we minimise the number of possible errors. And it feels right, we just don't reach the code we are not making use of anyways. 
And in terms of ff - agree, can be tricky. What helps me sometimes is just to write two describe blocks - one for ff enabled and one for ff disabled. It's weird that the ff is not reachable here 🤔 
But you have a valid point, I'm okay with keeping it as it is